internal/config.go

```diff
@@ -0,0 +1,66 @@
+package internal
```

Comment:
This file should use the new technique of loading a configuration file. See [this](https://github.com/Floor-Gang/authserver/blob/production/internal/config.go#L21) as an example.